src/EditorFeatures/Core/Implementation/Suggestions/SuggestedAction.cs

```diff
@@ -116,8 +117,14 @@ protected SolutionPreviewResult GetPreviewResult(CancellationToken cancellationT
         {
             if (_operations == null)
             {
-                _operations = Task.Run(
+                var extensionManager = this.Workspace.Services.GetService<IExtensionManager>();
```

Comment:
This seems unnecessary. GetPreviewResult() is already being called from within extensonManager.PerformFunction below. We don't need to wrap this twice.